### DISCUSSION TITLE
Is it possible to transcribe audio from direct download links instead of audio files?

### DISCUSSION THREAD

**Discussion by anserghazi** [ggml-org/whisper.cpp#595] [General]
I want to transcribe YouTube videos, but instead of using their audio files for whisper, is it possible to just refer to the direct link of the mp3? 

I use "yt-dlp -g [youtube video link]" to get the direct link, it usually begins with "https://rr7---sn-cxaaj5o5q5-tt1y.googlevideo.com/videoplayback?expire..."

**Comment by ArthurPeabody** [comment]
Whisper transcribes only WAV files, thus not.  Not only will you have to download the file, you'll have to strip off the video and convert the audio to WAV.  I use ffmpeg for both.

**Reply by anserghazi** [reply]
Thank you!
